boards/xtensa/esp32/esp32-devkitc/scripts/esp32.template.ld

```diff
@@ -50,7 +50,7 @@ MEMORY
    * be used.
    */
 
-  dram0_0_seg (RW) :     org = 0x3ffb0000 + CONFIG_ESP32_BT_RESERVE_DRAM,
+  dram0_0_seg (RW) :     org = 0x3ffae6f0 + CONFIG_ESP32_BT_RESERVE_DRAM,
```

Comment:
is there public doc or code from which one can infer these values?

Reply:
Please see https://github.com/espressif/esp-idf/blob/master/components/soc/esp32/soc_memory_layout.c#L168
IDF uses this as a seperate heap region, but because it's adjacent to 0x3ffb0000 I couldn't think of any reason why not extend static memory.

Reply:
unfortunately i don't have any better name suggestion.